helper/Data.php

```diff
@@ -100,6 +108,43 @@ public function getOriginKeyForOrigin()
         return $originKey;
     }
 
+    /**
+     * @param $store
+     * @param $country
+     * @return array
+     */
+    public function fetchPaymentMethods($countryCode, $amount, $currency, $shopperReference, $shopperLocale)
```

Comment:
Most of these parameters are in the Cart object. Passing the Cart object here would make calling this object less complex.